test/benchee/benchmark/runner_test.exs

```diff
@@ -282,228 +301,322 @@ defmodule Benchee.Benchmark.RunnerTest do
         ]
       }
 
-      %Suite{scenarios: [scenario]} = suite
-                                 |> test_suite
-                                 |> Benchmark.measure(TestPrinter)
+      %Suite{scenarios: [scenario]} =
+        suite
+        |> test_suite
+        |> Benchmark.measure(TestPrinter)
 
       # our previous run time isn't there anymore
       refute Enum.member?(scenario.run_times, 1_000_000)
     end
 
     test "global hooks triggers" do
       me = self()
+
       %Suite{
         configuration: %{
           warmup: 0,
           time: 100,
-          before_each: fn(input) -> send(me, :before); input end,
-          after_each: fn(_) -> send(me, :after) end
+          before_each: fn input ->
+            send(me, :before)
+            input
+          end,
+          after_each: fn _ -> send(me, :after) end
         }
       }
       |> test_suite
-      |> Benchmark.benchmark("job", fn -> :timer.sleep 1 end)
+      |> Benchmark.benchmark("job", fn -> :timer.sleep(1) end)
       |> Benchmark.measure(TestPrinter)
 
-      assert_received_exactly [:before, :after]
+      assert_received_exactly([:before, :after])
     end
 
     test "scenario hooks triggers" do
       me = self()
+
       %Suite{
         configuration: %{
           warmup: 0,
           time: 100
         }
       }
       |> test_suite
-      |> Benchmark.benchmark("job", {
-           fn -> :timer.sleep 1 end,
-           before_each: fn(input) -> send(me, :before); input end,
-           after_each: fn(_) -> send(me, :after) end,
-           before_scenario: fn(input) -> send(me, :before_scenario); input end,
-           after_scenario: fn(_) -> send(me, :after_scenario) end})
+      |> Benchmark.benchmark(
+        "job",
+        {fn -> :timer.sleep(1) end,
+         before_each: fn input ->
+           send(me, :before)
+           input
+         end,
+         after_each: fn _ -> send(me, :after) end,
+         before_scenario: fn input ->
+           send(me, :before_scenario)
+           input
+         end,
+         after_scenario: fn _ -> send(me, :after_scenario) end}
+      )
       |> Benchmark.measure(TestPrinter)
 
-      assert_received_exactly [
-        :before_scenario, :before, :after, :after_scenario
-      ]
+      assert_received_exactly([
+        :before_scenario,
+        :before,
+        :after,
+        :after_scenario
```

Comment:
we aren't allowed to put all these in a line anymore? :'(

Reply:
We can put them in one line, but because we had it over multiple lines before it did this instead of pulling it all up to a single line.